src/app/routes/applications/governance-list/governance-list.component.ts

```diff
@@ -368,7 +370,27 @@ export class GovernanceListComponent implements OnInit, OnDestroy {
     this.dataSource.data = JSON.parse(JSON.stringify(this.origDatasource));
   }
 
+  createSubOrg(parentOrg: any) {
+    const dialogRef = this.dialog.open(NewOrganizationComponent, {
+      width: '600px',
+      data: {
+        viewType: ViewType.NEW,
+        parentOrg: JSON.parse(JSON.stringify(parentOrg)),
+        owner: parentOrg.owner
+      },
+      maxWidth: '100%',
+      disableClose: true
+    });
+
+    dialogRef.afterClosed()
+      .pipe(
+        filter(Boolean),
+        takeUntil(this.subscription$))
+      .subscribe(async () => await this.newSubOrg(parentOrg));
+  }
+
   async newSubOrg(parentOrg: any) {
```

Comment:
I think that `newSubOrg` and `createSubOrg` are a bit too similar a name.  How about this rename?
```suggestion
  async viewNewSubOrg(parentOrg: any) {
```

Reply:
Ok, sure i can rename it. I just wanted to remove regression before deploying to volta and handle it in next few days properly.